engine/apps/alerts/tasks/check_escalation_finished.py

```diff
@@ -127,6 +133,14 @@ def check_escalation_finished_task() -> None:
         except AlertGroupEscalationPolicyExecutionAuditException:
             alert_group_ids_that_failed_audit.append(str(alert_group.id))
 
+    failed_alert_groups_count = len(alert_group_ids_that_failed_audit)
+    success_ratio = (
+        100
+        if total_alert_groups_count == 0
+        else (total_alert_groups_count - failed_alert_groups_count) / total_alert_groups_count * 100
+    )
+    task_logger.info(f"Alert group notifications success ratio: {success_ratio}")
```

Comment:
Makes sense to me. We should get a value for this every ~15mins corresponding to the previous 48hs, right?

Reply:
yes